integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -973,12 +975,10 @@ private static boolean isLocalPortFree(int port) {
     } catch (IOException ignored) {
       return true;
     } finally {
-      if (socket != null) {
-        try {
-          socket.close();
-        } catch (IOException ex) {
-          logger.severe("can not close Socket {0}", ex.getMessage());
-        }
+      try {
```

Comment:
Got a feedback from the team to try to close the socket without a null check.

Reply:
This isn't the right pattern. You want: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Put the socket constructor in the try statement.

Reply:
Good suggestion. fixed it.